cmd/tetragon/main.go

```diff
@@ -109,9 +109,16 @@ func getFieldFilters() ([]*tetragon.FieldFilter, error) {
 	return filters, nil
 }
 
-func getRedactionFilters() (fieldfilters.RedactionFilterList, error) {
+func setRedactionFilters() error {
+	var err error
 	redactionFilters := viper.GetString(option.KeyRedactionFilters)
```

Comment:
I think/believe finding a way to push these through a CRD would be more dynamic/easier to manage. But would be curious what experts managing clusters today think. For this PR lets just continue to use config like done here.